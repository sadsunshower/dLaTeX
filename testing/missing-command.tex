\documentclass{article}

\begin{document}

Test: missing command

The missing command is \texit{aaaaa} right here

\end{document}