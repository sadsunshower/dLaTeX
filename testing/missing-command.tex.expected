This is dLaTeX version 0.1
{"messages": [{"severity": "error", "message": "Unrecognised LaTeX command / macro. Did you mistype, or forget a package?", "line": "The missing command is \\texit{aaaaa} right here", "num": 7, "marker": [28]}], "error": true}
